azure-kusto-data/azure/kusto/data/kcsb.py

```diff
@@ -129,7 +129,7 @@ def __init__(self, connection_string: str):
             key, _, value = kvp_string.partition("=")
             keyword = self.ValidKeywords.parse(key)
             if keyword.is_str_type():
-                self[keyword] = value.strip()
+                self[keyword] = value.strip()[:-1] if value.strip()[-1] == "/" else value.strip()
```

Comment:
This way you're calling strip() twice, better to call once and then remove the slash